ReactAndroid/src/main/java/com/facebook/react/views/scroll/ReactScrollView.java

```diff
@@ -233,55 +239,40 @@ public void getClippingRect(Rect outClippingRect) {
 
   @Override
   public void fling(int velocityY) {
-    if (mScroller != null) {
```

Comment:
Moved down to if mPagingEnabled is false as fling is not needed if paging is enabled